### DISCUSSION TITLE
Send in specific time

### DISCUSSION THREAD

**Discussion by veselyslon** [Extended-Thunder/send-later#405] [Ideas]
I would like the opportunity not only to send an email with a delay, but to send it automatically at a defined time. That is, not the send button in 30 minutes, but send in 14:55. Therefore, I often send e-mails that I prepare throughout the day.

**Comment by jph-sendlater** [comment]
Can you explain what you mean a little more. Is this not something you can do by just typing "14:55" into the "Send at: _____" field?

**Comment by veselyslon** [comment]
yes, its possible. But i want change ie button "30 minutes from now" to "send in 14:55", so that I don't must fill time manually every time

**Reply by jph-sendlater** [reply]
It is possible to change those buttons, but an easier approach would be to enter "14:55" in the "send at ___" box, then use the "Save these values as defaults" button. That way, when you open the send later dialog it's already filled in

**Comment by lukas-schnieper** [comment]
Go to the extension settings and create a dynamic function, add the following code. It will set the time to 14:55 on the current day:

`next = new Date();
next.setHours(14,55,0,0);`

Than you can change one of the shortcuts above to your new dynamic function.
